test/presenters/return-requirements/start-date.presenter.test.js

```diff
@@ -7,74 +7,89 @@ const Code = require('@hapi/code')
 const { describe, it, beforeEach } = exports.lab = Lab.script()
 const { expect } = Code
 
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
```

Comment:
I could have missed something. But do we need to create a session record in the DB for these unit tests? Could we not stick with the strategy of just passing in an object that represents an instance of `SessionModel`?

Reply:
We don't I've removed this and turned into a simple object now